src/main/java/com/gooddata/md/visualization/VisualizationObject.java

```diff
@@ -281,7 +300,11 @@ public ResultSpec convertToResultSpec(Function<String, VisualizationClass> visua
         return VisualizationConverter.convertToResultSpec(this, visualizationClassgetter);
     }
 
-    private Content getContent() {
+    /**
+     * Returns content of this visualization object.
+     * Mainly for serialization purposes.
+     */
+    public Content getContent() {
```

Comment:
Jackson can serialize even from protected methods, so do we want to expose it to public?

Reply:
fixed